lib/db/schema/patch-068-067.sql

```diff
@@ -0,0 +1,14 @@
+-- ALTER TABLE `unverifiedTokens`
+-- DROP COLUMN `shortCode` BINARY(16),
+-- ALGORITHM = INPLACE, LOCK = NONE;
+
+-- ALTER TABLE `unverifiedTokens`
+-- DROP COLUMN `shortCodeExpiresAt` BIGINT UNSIGNED,
+-- ALGORITHM = INPLACE, LOCK = NONE;
```

Comment:
Should we also drop the `unverifiedTokens_shortCodeHash` index here, potentially before removing the columns in question?